Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -142,20 +142,20 @@ namespace Babylon
         void UseViewId(uint16_t viewId)
         {
             ViewId = viewId;
-            ViewClearState.UpdateViewId(ViewId);
+            viewClearState.UpdateViewId(ViewId);
         }
 
         void SetUpView(uint16_t viewId)
         {
             UseViewId(viewId);
             bgfx::setViewFrameBuffer(ViewId, FrameBuffer);
-            ViewClearState.Update();
+            viewClearState.Update();
             bgfx::setViewRect(ViewId, 0, 0, Width, Height);
         }
 
         bgfx::FrameBufferHandle FrameBuffer{bgfx::kInvalidHandle};
         bgfx::ViewId ViewId{};
-        ViewClearState ViewClearState;
+        ViewClearState viewClearState;
```

Comment:
Maybe rename the type instead of the variable. Otherwise, the variable conventions are mixed.